Add support for sigv4 in metamonitoring (#9279)
grafana/mimir: operations/helm/charts/mimir-distributed/CHANGELOG.md

* [BUGFIX] Alertmanager: Set -server.http-idle-timeout to avoid EOF errors in ruler. #8192
* [BUGFIX] Helm: fix second relabeling in ServiceMonitor and PVC template in compactor to not show diff in ArgoCD. #9195
* [BUGFIX] Helm: create query-scheduler `PodDisruptionBudget` only when the component is enabled. #9270

> * [ENHANCEMENT] Add support for sigv4 for metamonitoring. Allows you to connect metamonitoring to AMP #9279

Reviewer: ```suggestion
* [ENHANCEMENT] Add support for sigv4 authentication for remote write in metamonitoring. #9279
```
